README.md

```diff
@@ -1,4 +1,6 @@
 # Icicle [![Build Status](https://travis-ci.org/intenthq/icicle.svg?branch=master)](https://travis-ci.org/intenthq/icicle)
+
+[![Join the chat at https://gitter.im/intenthq/icicle](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/intenthq/icicle?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Could you move this to the same line as the travis badge (so they're side by side)?